meteor/client/ui/Settings/Studio/Routings.tsx

```diff
@@ -12,723 +11,903 @@ import {
-
-						const routeMappingSchema = manifest[(routeDeviceType ?? route.remapping?.device) as TSR.DeviceType]
-
-						const rawMappingTypeOptions = Object.entries<JSONSchema>(routeMappingSchema?.mappingsSchema || {})
-						const mappingTypeOptions = rawMappingTypeOptions.map(([id, entry], i) =>
-							literal<DropdownInputOption<string | number>>({
-								value: id + '',
-								name: entry?.title ?? id + '',
-								i,
-							})
-						)
-
-						return (
-							<div className="route-sets-editor mod pan mas" key={index}>
-								<button
-									className="action-btn right mod man pas"
-									onClick={() => this.confirmRemoveRoute(routeSetId, route, index)}
-								>
-									<FontAwesomeIcon icon={faTrash} />
-								</button>
-								<div className="properties-grid">
-									<label className="field">
-										<LabelActual label={t('Original Layer')} />
-										<EditAttribute
-											modifiedClassName="bghl"
-											attribute={`routeSets.${routeSetId}.routes.${index}.mappedLayer`}
-											obj={this.props.studio}
-											type="dropdowntext"
-											options={Object.keys(this.props.studioMappings)}
-											label={t('None')}
-											collection={Studios}
-											className="input text-input input-l"
-										></EditAttribute>
-									</label>
-									<label className="field">
-										<LabelActual label={t('New Layer')} />
-										<EditAttribute
-											modifiedClassName="bghl"
-											attribute={`routeSets.${routeSetId}.routes.${index}.outputMappedLayer`}
-											obj={this.props.studio}
-											type="text"
-											collection={Studios}
-											className="input text-input input-l"
-										></EditAttribute>
-									</label>
-
-									<label className="field">
-										<LabelActual label={t('Route Type')} />
-										{!route.mappedLayer ? (
-											<span className="mls">REMAP</span>
+					<p className="mhn mvs text-s dimmed field-hint">
+						{t(
+							'Controls for exposed Route Sets will be displayed to the producer within the Rundown View in the Switchboard.'
+						)}
+					</p>
+					<h3 className="mhn">{t('Exclusivity Groups')}</h3>
+					<table className="expando settings-studio-mappings-table">
+						<tbody>
+							<RenderExclusivityGroups
+								studio={studio}
+								getRouteSetsFromOverrides={getRouteSetsFromOverrides}
+								isExpanded={isExpanded}
+								toggleExpanded={toggleExpanded}
+								getExclusivityGroupsFromOverrides={getExclusivityGroupsFromOverrides}
+							/>
+						</tbody>
+					</table>
+					<div className="mod mhs">
+						<button className="btn btn-primary" onClick={() => addNewExclusivityGroup()}>
+							<FontAwesomeIcon icon={faPlus} />
+						</button>
+					</div>
+					<h3 className="mhn">{t('Route Sets')}</h3>
+					<table className="expando settings-studio-mappings-table">
+						<tbody>
+							{_.map(getRouteSetsFromOverrides, (routeSet: WrappedOverridableItem<StudioRouteSet>) => {
```

Comment:
It would be nice to replace the `_.map` with native methods, but it is not essential. 
NRK has (or had?) a very low priority aim to replace usage of underscore with native methods whenever possible

Reply:
Nice to know, I was also wondering why it still was used in the code.
Wil change that